test/query.test.js

```diff
@@ -1076,16 +1076,12 @@ describe('query', () => {
         var documentRef = result.ref
         return client
           .query(query.Login(documentRef, { password: 'sekrit' }))
-          .then(function(result2) {
+          .then(async function(result2) {
```

Comment:
Not ES5 compatible :(

Reply:
It is only for tests. No production code at all.

Reply:
👍